lib/zendesk_api/collection.rb

```diff
@@ -324,22 +324,18 @@ def to_param
       map(&:to_param)
     end
 
-    def has_more_results?(response)
-      return false unless response["meta"].present? && response["results"].present?
-
-      response["meta"]["has_more"] && response["results"].length > 0
+    def more_results?(response)
+      response["meta"].present? && response["results"].present?
```

Comment:
I'm quite confident these two versions are equivalent, I don't mind a second opinion.

Reply:
I am thinking that the name should contain `has` as it is a boolean function. Any particular reason for name change?

Reply:
this change is due to a Ruby convention, enforced by rubocop. Methods ending in `?` don't need a prefix `is` or `has`.